modules/process-monitor/Cargo.toml

```diff
@@ -12,6 +12,7 @@ tokio = { version = "1", features = ["full"] }
 nix = "0.24.0"
 log = "0.4"
 thiserror = "1"
+which = "4.2.5"
```

Comment:
This should go under `[dev-dependencies]`

Reply:
done in d15bc0a